apps/src/utils.js

```diff
@@ -97,7 +98,7 @@ export function range(start, end) {
  * Given two functions, generates a function that returns the result of the
  * second function if and only if the first function returns true
  */
-export function executeIfConditional(conditional, fn) {
+export function executeIfConditional(conditional:()=>bool, fn:Function):Function {
```

Comment:
it's not so bad once you get used to it. I think it's actually pretty nice for callbacks where you really want to specify the function signature that you are expecting the callback to have.